auto_nag/scripts/assignee_no_login.py

```diff
@@ -110,9 +153,6 @@ def get_bz_params(self, date):
         params = {
             "include_fields": fields,
             "resolution": "---",
-            "f1": "assignee_last_login",
-            "o1": "lessthan",
-            "v1": start_date,
```

Comment:
How long is the query taking now? It is super broad, maybe we should exclude recently touched bugs?

Reply:
> How long is the query taking now? 

It does not take much, based on the dryrun above, it took 48 seconds. 

> It is super broad, maybe we should exclude recently touched bugs?

We already exclude bugs that got assigned recently (not general activities):

https://github.com/mozilla/relman-auto-nag/blob/5c7aab92e94d14f61e46f80cebd96dcedca92fdf/auto_nag/scripts/assignee_no_login.py#L157-L159

Reply:
All right, 48 seconds is acceptable.